src/App.tsx

```diff
@@ -10,33 +12,39 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState<string>(tabs[0].id);
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState<string>(tabs[0]);
```
you can set the whole tab to state to avoid calling filter to find the title